docs/words.tex

```diff
@@ -356,9 +356,20 @@ \section{System State}
 
 \begin{description}
 
-\item[latest (variable)] Position of latest defined word.
 
-\item[here (variable)] Write position of the Forth compiler (usually first unused byte of memory). Many C64 assemblers refer to this as program counter or \texttt{*}.
+\item[latest (variable)] Address of latest defined header.

```

Comment:
I suppose "header" is now actually a little misleading, since the word is split there is no longer a head or tail. "Wordlist entry"? But it is splitting hairs, we still have the word HEADER , so it keeping it is OK I guess.